dev-python/nuitka/nuitka-0.6.0.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_5 python3_6 )
+inherit distutils-r1
+
+DESCRIPTION="Python to native compiler"
+HOMEPAGE="http://www.nuitka.net"
+SRC_URI="http://nuitka.net/releases/${PN^}-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	>=dev-python/appdirs-1.4.3
```

Comment:
Why no PYTHON_USEDEP?

Reply:
just because the Debian package was so specific, together with my noobness with Python ebuilds. I now changed version 0.6.0.4 to include PYTHON_USEDEP.

Reply:
see version 0.6.0.4 pushed below, I removed 0.6.0.2, the commit history to includes only version 0.6.0.4 now.